roles/openshift_manage_node/tasks/main.yml

```diff
@@ -67,11 +67,17 @@
   delegate_to: "{{ openshift_master_host }}"
 
 - name: Label nodes
-  command: >
-    {{ hostvars[openshift_master_host].openshift.common.client_binary }} label --overwrite node {{ openshift.node.nodename }} {{ openshift.node.labels | oo_combine_dict  }}
-    --config={{ openshift_manage_node_kubeconfig }}
-    -n default
-  when: "'nodename' in openshift.node and 'labels' in openshift.node and openshift.node.labels != {}"
+  oc_label:
+    name: "{{ openshift.node.nodename }}"
+    kind: node
+    state: add
+    labels: "{{ openshift.node.kubelet_args.node-labels }}"
```

Comment:
@detiber  What's the different between openshift.node.kublet_args.node-labels and openshift.node.labels?  The first is a list and the second is a dict.  I need a list so would like to use kublet_args.node-labels. Instead of taking the dict from openshift.node.labels and transforming it to a string using oo_combine_dict and then converting it to a list of oc_label, I just wanted to use the list that is already in the form needed. http://pastebin.test.redhat.com/454366